tests/unit/EntityMenu.spec.ts

```diff
@@ -499,9 +501,8 @@ describe('Entity Menu - Request AGM Extension click tests', () => {
       mixins: [{ methods: { isAllowed: () => true } }],
       propsData: { businessId: 'BC1234567' }
     })
-    vm = wrapper.vm
 
-    vi.spyOn(vm, 'enableAgmLocationChg', 'get').mockReturnValue(CorpTypeCd.BENEFIT_COMPANY)
+    vm = wrapper.vm
     vi.spyOn(vm, 'enableAgmExtension', 'get').mockReturnValue(true)
```

Comment:
You probably don't need this since the default flag (line 487) is True. So maybe the default flags should be False and "".